src/tasks/mod.rs

```diff
@@ -139,13 +141,45 @@ where
     let shut_down = Arc::new(AtomicBool::new(false));
     let started = Arc::new(AtomicBool::new(false));
 
+    let exchange = hotshot.inner.quorum_exchange.clone();
+    let committee_exchange = hotshot.inner.committee_exchange.clone();
+
+    let _network_broadcast_task_handle = async_spawn(
+        network_task(
+            hotshot.clone(),
+            shut_down.clone(),
+            TransmitType::Broadcast,
+            exchange.clone(),
+        )
+        .instrument(info_span!("HotShot Broadcast Task",)),
+    );
+    let _network_direct_task_handle = async_spawn(
+        network_task(
+            hotshot.clone(),
+            shut_down.clone(),
+            TransmitType::Direct,
+            exchange,
+        )
+        .instrument(info_span!("HotShot Direct Task",)),
+    );
```

Comment:
Just double-checking, do we plan to add these to the `HotShotInner` in the future? Not sure I understand why we add the DA tasks but not these two quorum tasks.

Reply:
It should be in `TaskHandleInnder` I'll add it there.  It's used for shutdown so I will add it there now.